File: AKLanguageManager/iOS/DemoObjC/DemoObjC/FirstViewController.m
//
//  FirstViewController.m
//  DemoObjC
//
//  Created by Amr Koritem on 13/10/2022.
//

#import "FirstViewController.h"
@import AKLanguageManager;

@interface FirstViewController ()

@property (weak, nonatomic) IBOutlet UIImageView *imageView;

@end

@implementation FirstViewController

- (IBAction)goToSettings {
    UIViewController* settingsViewController = [self.storyboard instantiateViewControllerWithIdentifier:@"SettingsViewController"];
    [self.navigationController pushViewController:settingsViewController animated:true];
}

- (IBAction)switchImageDirection {
    [_imageView revertImageHorizontalDirection];
}

- (void)viewDidLoad {
    [super viewDidLoad];
    self.title = @"Main".localized;
}


@end

File: AKLanguageManager/tvOS/DemoUIKit/DemoUIKit/FirstViewController.swift
//
//  FirstViewController.swift
//  DemoUIKit
//
//  Created by Amr Koritem on 19/10/2022.
//

import UIKit
import AKLanguageManager

class FirstViewController: UIViewController {
    @IBOutlet weak var imageView: UIImageView!

    // Changing the language from another screen.
    @IBAction func goToSettings() {
        let settingsVC = Storyboard.Main.instantiate(viewController: SettingsViewController.self)
        show(settingsVC, sender: self)
    }

    // Change the image direction while the selected language doesn't change.
    @IBAction func switchImageDirection() {
        imageView.revertImageHorizontalDirection()
    }

    override func viewDidLoad() {
        super.viewDidLoad()
        title = "Main".localized
    }
}
